src/math/p5.Vector.js

```diff
@@ -1572,53 +1777,77 @@ p5.Vector = class {
   }
 
   /**
- * Calculates and returns the angle between two vectors. This method will take
- * the current <a href="#/p5/angleMode">angleMode</a> into consideration, and
- * give the angle in radians or degrees accordingly.
+ * Returns the angle between two vectors.
```

Comment:
I don't think we mention this anywhere but we return a signed angle, so if `v1.angleBetween(v2)` is positive, then I think `v2.angleBetween(v1)` is negative. Should we mention that so people know to handle it, or add `abs()` if they don't want that?

Reply:
Good suggestion. Let me know how this draft reads.